server/monitor_test.go

```diff
@@ -4212,13 +4212,29 @@ func TestMonitorJsz(t *testing.T) {
 				t.Fatalf("expected single account")
 			}
 			acc := info.AccountDetails[0]
-			got := acc.ReservedMemory
-			expected := uint64(5242880)
+			got := int(acc.ReservedMemory)
+			expected := 5242880
 			if got != expected {
 				t.Errorf("Expected: %v, got: %v", expected, got)
 			}
-			got = acc.ReservedStore
-			expected = uint64(4194304)
+			got = int(acc.ReservedStore)
+			expected = 4194304
+			if got != expected {
+				t.Errorf("Expected: %v, got: %v", expected, got)
+			}
+
+			info = readJsInfo(url + "?accounts=true&acc=BCC_TO_HAVE_ONE_EXTRA")
+			if len(info.AccountDetails) != 1 {
+				t.Fatalf("expected single account")
+			}
+			acc = info.AccountDetails[0]
+			got = int(acc.ReservedMemory)
```

Comment:
Again, would keep got as acc.ReservedMemory, but instead have `expected = uint64(-1)`.

Reply:
Compiler does not let you do uint64(-1), so disregard.